packages/api/models/PortfolioDraftSummary.ts

```diff
@@ -12,3 +10,15 @@ export interface PortfolioDraftSummary extends BaseDocument {
   num_applications: number;
   num_environments: number;
 }
+
+export const portfolioDraftFields: ExhaustiveAttributeMap<PortfolioDraftSummary> = {
+  id: null,
+  created_at: null,
+  updated_at: null,
+  status: null,
+  name: null,
+  num_portfolio_managers: null,
+  num_task_orders: null,
+  num_applications: null,
+  num_environments: null,
```

Comment:
If you play around with this locally and try to remove or add items to this list (without also changing the interface), you'll see that you get TypeScript errors

Reply:
Is it possible to do
```
  num_portfolio_managers?: int,
  num_task_orders?: int,
  num_applications?: int,
  num_environments?: int,

Reply:
@zachclark-ccpo `number` is a type, so not really. `null` is a value and it's a value that we currently don't apply any meaning to (and is friendlier than `null` or `undefined`).